faq.html

```diff
@@ -155,10 +155,13 @@ <h2 id="general">General</h2>
       <a href="{{page.root}}/img/software-carpentry-banner.png">Software Carpentry logo</a>
       are both trademarked,
       and may only be used with our explicit prior permission.
-      We normally grant this to any training course that covers our
-      <a href="#core-topics">core topics</a>
-      has at least one <a href="{{page.root}}/pages/team.html">certified instructor</a> teaching,
-      and sends us summary information about attendees.
+      We normally grant this to any training course that:
```

Comment:
@karinlag They have to be compliant with some specifications. Obviously, it's unlikely someone will create their own version of the template), however I think that @r-gaia-cs suggestion is a good idea - we should make it explicit to the self-organised workshops what we want from them.